testing_and_setup/atmosphere/setup_atm_run_dir

```diff
@@ -0,0 +1,175 @@
+    if [ $? -ne 0 ]; then
+        printf "Failed to copy 'namelist.init_atmosphere' from %s\n" "${mpasdir}/default_inputs"
+        return 1
+    fi
+
+    cp "${mpasdir}/default_inputs/streams.init_atmosphere" $rundir
+    if [ $? -ne 0 ]; then
+        printf "Failed to copy 'streams.init_atmosphere' from %s\n" "${mpasdir}/default_inputs"
+        return 1
+    fi
+
+    printf "Succesfully setup the run directory for the init_atmosphere model\n"
+    return 0
+}
+
+######################################################################
+# atmosphere_setup()
+#
+# $1 - Run directory
+# $2 - MPAS-Model Source Code Directory
+# Setup the atmosphere core in the run directory ($1) by linking the
+# atmosphere_model exeutable, physics lookup tables, and by copying
+# the needed namelist, streams, and stream_lists from the default_inputs/
+# directory.
+#
+# On error, this function -1 will be returned, otherwise, 0 will be
+# returned.
+#
+######################################################################
+atmosphere_setup()
+{
+    printf "Setting up the atmosphere core ...\n"
+    rundir=$1
+    mpasdir=$2
+
+    # See if the amtosphere core is compiled
+    if ! [ -f "${mpasdir}/atmosphere_model" ]; then
+        printf "The MPAS directory does not appear to have the atmosphere core compiled!\n"
+        return 1
+    fi
+
+    ln -s "${mpasdir}/atmosphere_model" $rundir
+    if [ $? -ne 0 ]; then
+        printf "Failed to link 'atmosphere_model' from %s\n" $mpasdir
+        return 1
+    fi
+
+    cp "${mpasdir}/default_inputs/namelist.atmosphere" $rundir
+    if [ $? -ne 0 ]; then
+        printf "Failed to copy 'namelist.atmosphere' from %s\n" "${mpasdir}/default_inputs"
+        return 1
+    fi
+
+    cp "${mpasdir}/default_inputs/streams.atmosphere" $rundir
+    if [ $? -ne 0 ]; then
+        printf "Failed to copy 'streams.atmosphere' from %s\n" "${mpasdir}/default_inputs"
+        return 1
+    fi
+
+    cp ${mpasdir}/default_inputs/stream_list.atmosphere.* $rundir
+    if [ $? -ne 0 ]; then
+        printf "Failed to copy 'stream_list.atmosphere.*' from %s\n" "${mpasdir}/default_inputs"
+        return 1
+    fi
+
+    ln -s ${mpasdir}/src/core_atmosphere/physics/physics_wrf/files/* $rundir
+    if [ $? -ne 0 ]; then
+        printf "Failed to link physics files from %s\n" "${mpasdir}/src_core_atmosphere_physics/physics_wrf/files"
+        return 1
+    fi
+
+    printf "Succesfully setup the run directory for the atmosphere model\n"
+    return 0
+}
+
+
+########################################################
+#
+# setup_run_atm_run_dir.sh
+#
+# \brief Copy and link the needed files for running the init_atmosphere, and
+# atmosphere core.
+# \details
+# Given a directory, copy and/or link all the needed executables, namelist,
+# streams, stream_lists and physics lookup tables needed for both the
+# init_atmosphere and atmosphere core.
+#
+# Currently, this script will need to be called from the
```

Comment:
This is no longer true, right?

Reply:
I think I had the wrong wording here to begin with. I think what I meant to say was that the script needed to be _in_ the `testing_and_setup/atmosphere/` directory, rather than _called_. I've updated this comment to be clearer.